libsolidity/formal/CHC.h

```diff
@@ -121,9 +121,6 @@ class CHC: public SMTEncoder
 	std::vector<smt::Expression> currentBlockVariables();
 
 	/// Sets the SSA indices of the variables in scope to 0.
-	/// Used when starting a new block.
-	void clearIndices();
```

Comment:
Moved to `SMTEncoder`

Reply:
There's one line of comment still there that should also be removed, isn't there?